packages/crashlytics/ios/Plugin/FirebaseCrashlyticsPlugin.swift

```diff
@@ -108,20 +111,39 @@ public class FirebaseCrashlyticsPlugin: CAPPlugin, CAPBridgedPlugin {
     }
 
     @objc func recordException(_ call: CAPPluginCall) {
-        guard let message = call.getString("message") else {
-            call.reject(errorMessageMissing)
-            return
+      guard let message = call.getString("message") else {
+        call.reject(errorMessageMissing)
+        return
+      }
+      
+      let stacktraceOptional = call.getArray("stacktrace")
+      let customPropertiesOptional = call.getArray("customProperties")
+        
+      guard !(stacktraceOptional != nil && customPropertiesOptional != nil) else {
+        call.reject(errorStacktraceAndCustomPropertiesDisallowed)
+        return
+      }
+      
+      if(stacktraceOptional != nil) {
+        guard let stacktrace = stacktraceOptional as? [JSObject] else {
+          call.reject(errorStacktraceCastFailed)
+          return
```

Comment:
Please remove this check as this shouldn't be part of this PR. The behavior should just be the same as before.